cypress/integration/my_cards_spec.js

```diff
@@ -10,10 +10,9 @@ describe('The Player Cards Component', () => {
         });
     });
 
-    /* TODO: figure out how to test border */
     describe('player clicks on a card', () => {
         it("its highlighted by being moved upwards on the y axis", () => {
-            cy.playCard();
+            cy.get('[data-cy="my-cards"] [data-cy="card"]').first().click();
```

Comment:
how come you're not using one of the commands?

Reply:
The command is for actually sending the card, whereas this test is for seeing whether it becomes 'selected' when you click on it